src/plugin/isToday.ts

```diff
@@ -1,19 +1,20 @@
 import type { Dayjs, Plugin } from '..'
 
-export type IsToday = (this: Dayjs) => boolean
+export type isTodayType = () => boolean
```

Comment:
IMHO, the name of typing should be `IsToday`.

Reply:
As this is an interface, you are of right - my fault.